training/DeepSpeed-Domino/requirements.txt

```diff
@@ -0,0 +1,6 @@
+apex
+deepspeed
```

Comment:
Remember to update this like below to match the deepspeed release. 
```suggestion
deepspeed>=version
```

Reply:
Sure, the release version is still uncertain. I'll provide an update once domino is merged into deepspeed.